src/components/Card/mixins.st.css

```diff
@@ -0,0 +1,16 @@
+:vars {
+    /*Overrides*/
+    BorderWidth: --overidable;
+    BorderColor: --overidable;
+    PaddingSize: --overidable;
+}
+
+.withBorder {
+    border-style: solid;
+    border-color: value(BorderColor);
+    border-width: value(BorderWidth);
+}
+
+.withDynamicPadding {
+    padding: calc(value(PaddingSize) - value(BorderWidth));
```

Comment:
just be aware that this is a runtime `calc` function and not a static one.
if you do want a static one, you need to use a mixin like this
https://github.com/wix/wix-ui/blob/master/packages/wix-ui-core/src/mixins/calc/index.js

Reply:
yeah i'm a ware of it i need the padding to be responsive to the border width.
the border width is dynamically changes in the settings panel and the should
correspond to it